django/src/rdwatch/views/site_image.py

```diff
@@ -23,7 +23,7 @@ class SiteImageSchema(Schema):
     source: str
     cloudcover: float | None
     image: str
-    observation_id: UUID4 | None
+    observation_id: UUID4 | str | None
```

Comment:
Leftover from debugging?
```suggestion
    observation_id: UUID4 | None
```

Reply:
Yup